ddcommon-ffi/src/option.rs

```diff
@@ -31,3 +31,13 @@ impl<T: Copy> From<&Option<T>> for std::option::Option<T> {
         }
     }
 }
+
+#[no_mangle]
+pub extern "C" fn ddog_Option_U32_some(v: u32) -> Option<u32> {
+    Option::Some(v)
+}
+
+#[no_mangle]
+pub extern "C" fn ddog_Option_U32_none() -> Option<u32> {
+    Option::None
+}
```

Comment:
Minor: There's a bit of an extra overhead in having these be actual Rust code rather than something that only exists in the C headers as a macro/inline function but... I guess we can wait until we spot it as an actual source of overhead.

Reply:
👍